### ISSUE TITLE
Use HTML language tags for text that is not in the UI language

### ISSUE THREAD

**Issue by osma** [NatLibFi/Skosmos#1876]
## Description of the enhancement

Skosmos HTML pages set their language based on the UI language, but often text (labels, notes etc.) in other languages is displayed on those pages, especially when the content language is different from the UI language. This language mismatch may throw off screen readers and other software interpreting the HTML.

Skosmos HTML code elements should use the `lang` attribute to indicate the language whenever it differs from the UI language. This would give a hint to screen readers that the text within the element should be read in that language.

It would be better 

## Who are the users that would benefit from the enhancement and how?

Users of screen readers

## What new functionalities would the enhancement make possible?

Screen readers would read text correctly even if the text is not in the same language as the rest of the page.

## Why is the enhancement important?

Accessibility is important.

**Comment by osma** [comment]
The same idea was proposed earlier by @kouralex in #1057 

> If you use a screen reader and tab to the UI language selection you will hear the textual content in the same language as the main document. However, it should read it in the respective language, e.g., the Finnish text 'suomeksi' in Finnish etc.
>
>This can be achieved by adding the HTML lang attribute (with the value of fi in the above example) to the respective element.
>
>Please note that there are multiple places where fixes are needed. One that comes to my mind is when one has changed the content language (thus, the clang parameter) for a vocabulary, resulting in foreign language values displayed on the screen.
